### PR TITLE
login3 styling fixes

### PR DESCRIPTION
**Pull request by chrisnojima**
Addresses feedback from @cecileboucheron

### CODE REVIEW

File: shared/common-adapters/button.desktop.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ProgressIndicator from './progress-indicator'
 import {globalStyles, globalColors} from '../styles/style-guide'
 import type {Props} from './button'
 
-export default class Button extends Component {
+class Button extends Component {
```
Comment (chrisnojima) [inline shared/common-adapters/button.desktop.js]: let's start moving towards having exports at the bottom

File: shared/common-adapters/button.desktop.js [hunk 2]
```diff
@@ -144,66 +144,65 @@ const buttonCommon = {
   minWidth: 10
 }
 
-const styles = {
```
Comment (chrisnojima) [inline shared/common-adapters/button.desktop.js]: as mentioned in person let's move away from having this style object. eslint doesn't help us find orphaned unreferenced properties in here

File: shared/common-adapters/icon.desktop.js [hunk 3]
```diff
@@ -63,6 +63,11 @@ export default class Icon extends Component {
     let hoverColor = this._defaultHoverColor(this.props.type)
     let iconType = this._typeToIconMapper(this.props.type)
 
+    if (!iconType) {
```
Comment (chrisnojima) [inline shared/common-adapters/icon.desktop.js]: let's not crash if we don't pass in a valid icon

File: shared/common-adapters/icon.js.flow [hunk 4]
```diff
@@ -5,8 +5,11 @@ import React, {Component} from 'react'
 export type Props = {
   type: 'GPG-export'
     | 'computer-big'
+    | 'computer-bw-m'
```
Comment (chrisnojima) [inline shared/common-adapters/icon.js.flow]: new icon sizes

File: shared/login/register/gpg-sign/row.desktop.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ const realCSS = `
 
   .register-row:hover .register-icon { transform: translateX(15px)}
 
-  .register-row .register-background { background-color: ${globalColors.lightGrey2}; }
+  .register-row .register-background { background-color: ${globalColors.lightGrey}; }
```
Comment (chrisnojima) [inline shared/login/register/gpg-sign/row.desktop.js]: greys went from 3 primary ones to 1 primary and 2 secondary

File: shared/login/register/select-other-device/index.render.desktop.js [hunk 6]
```diff
@@ -9,9 +9,9 @@ import type {DeviceType} from '../../../constants/types/more'
 
 const Row = ({deviceID, name, type, onSelect}) => {
   const iconType: IconProps.type = ({
-    'mobile': 'fa-mobile',
-    'desktop': 'fa-laptop',
-    'backup': 'paper-key'
+    'mobile': 'phone-bw-m',
```
Comment (chrisnojima) [inline shared/login/register/select-other-device/index.render.desktop.js]: these got blown away when we changed the types, also now in a different size

File: shared/styles/style-guide-colors.js [hunk 7]
```diff
@@ -24,9 +24,9 @@ export default {
   yellowGreen: '#a8cf36',
   yellowGreen2: '#89a82c',
 
-  lightGrey: '#f6f6f6',
-  lightGrey2: '#ebebeb',
-  lightGrey3: '#e0e0e0',
+  lightGrey: '#f0f0f0',
```
Comment (chrisnojima) [inline shared/styles/style-guide-colors.js]: most everything is lightGrey now. lightGrey2 is used by the buttons, grey is used by the little followed bubbles

File: shared/common-adapters/button.desktop.js [hunk 8]
```diff
@@ -144,66 +144,65 @@ const buttonCommon = {
...
-    left: 0,
-    right: 0,
-    top: 0,
-    bottom: 0,
-    margin: 'auto'
-  }
+const stylesButtonPrimary = {
+  ...buttonCommon,
+  backgroundColor: globalColors.blue,
+  disabledOpacity: 0.2
```
Comment (chrisnojima) [inline shared/common-adapters/button.desktop.js]: marginRight: 10 removed here. i think this just causes problems, our center aligned buttons are slighly off, the stretched ones also. in practice there's nothing 'to the right' of the button so we don't really want this

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**